Disable network grabbers that need boost::asio signal_set (unsupported SA_RESTART on QNX)

--- a/io/CMakeLists.txt
+++ b/io/CMakeLists.txt
@@ -226,12 +226,8 @@
   src/obj_io.cpp
   src/ifs_io.cpp
   src/image_grabber.cpp
-  src/hdl_grabber.cpp
-  src/vlp_grabber.cpp
-  src/robot_eye_grabber.cpp
   src/auto_io.cpp
   src/io_exception.cpp
-  src/tim_grabber.cpp
   ${VTK_IO_SOURCE}
   ${OPENNI_GRABBER_SOURCES}
   ${OPENNI2_GRABBER_SOURCES}
@@ -276,12 +272,8 @@
   "include/pcl/${SUBSYS_NAME}/ascii_io.h"
   "include/pcl/${SUBSYS_NAME}/ifs_io.h"
   "include/pcl/${SUBSYS_NAME}/image_grabber.h"
-  "include/pcl/${SUBSYS_NAME}/hdl_grabber.h"
-  "include/pcl/${SUBSYS_NAME}/vlp_grabber.h"
-  "include/pcl/${SUBSYS_NAME}/robot_eye_grabber.h"
   "include/pcl/${SUBSYS_NAME}/point_cloud_image_extractors.h"
   "include/pcl/${SUBSYS_NAME}/io_exception.h"
-  "include/pcl/${SUBSYS_NAME}/tim_grabber.h"
   ${VTK_IO_INCLUDES}
   ${OPENNI_GRABBER_INCLUDES}
   ${OPENNI2_GRABBER_INCLUDES}
